package-lock.json

```diff
@@ -276,6 +276,14 @@
                 "@octokit/types": "^5.0.0",
                 "is-plain-object": "^5.0.0",
                 "universal-user-agent": "^6.0.0"
+            },
+            "dependencies": {
+                "is-plain-object": {
+                    "version": "5.0.0",
+                    "resolved": "https://registry.npmjs.org/is-plain-object/-/is-plain-object-5.0.0.tgz",
+                    "integrity": "sha512-VRSzKkbMm5jMDoKLbltAkFQ5Qr7VDiTFGXxYFXXowVj387GeGNOCsOH6Msy00SGZ3Fp84b1Naa1psqgcCIEP5Q==",
+                    "dev": true
+                }
```

Comment:
Different npm version or something? (We’re thinking about making @typescript-bot push commits reverting accidental package-lock changes.)

```suggestion
```